parsec/core/gui/login_widget.py

```diff
@@ -158,7 +174,9 @@ class LoginAccountsWidget(QWidget, Ui_LoginAccountsWidget):
     pending_finalize_clicked = pyqtSignal(PkiEnrollmentSubmitterAcceptedStatusCtx)
     pending_clear_clicked = pyqtSignal(BasePkiEnrollmentSubmitterStatusCtx)
 
-    def __init__(self, config, jobs_ctx, devices, pendings):
+    def __init__(
+        self, config: CoreConfig, jobs_ctx: Any, devices: list[AvailableDevice], pendings: list[Any]
```

Comment:
jobs_ctx is a QtToTrioJobSchedule
also `pendings` type is ` List[PkiEnrollmentSubmitterSubmittedCtx]:`